backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewEdgeToolSwitch.java

```diff
@@ -50,7 +50,7 @@
  */
 public class ViewEdgeToolSwitch extends SysmlEClassSwitch<List<EdgeTool>> {
 
-    private static final String METAMODEL_ICONS_PATH = "/icons/full/obj16/";
+    public static final String METAMODEL_ICONS_PATH = "/icons/full/obj16/";
```

Comment:
Do you really want to change the visibility? If so, you should probably move this constant into another class like `ViewConstants.java`